net-im/ejabberd/ejabberd-21.12-r2.ebuild

```
@@ -56,7 +56,7 @@ DEPEND=">=dev-lang/erlang-19.3[odbc?,ssl]
 	stun? ( >=dev-erlang/stun-1.0.47 )
 	zlib? ( >=dev-erlang/ezlib-1.0.10 )"
 RDEPEND="${DEPEND}
-	acct-user/ejabberd
+	acct-user/ejabberd[pam?]
```

```suggestion
	>=acct-user/ejabberd-2[pam?]
```

and if it is just to denote when the pam USE flag was introduced to the acct-user/ejabberd ebuild.

---

Done.